src/main/resources/config/tpcds/wp3_rw_concurrency_multi.yaml

```diff
@@ -0,0 +1,83 @@
+# Description: WP3: R/W concurrency (multi cluster)
+---
+version: 1
+id: wp3_rw_concurrency_multi
+phases:
+- id: setup
+  sessions:
+  - tasks:
+    - template_id: setup
+- id: setup_data_maintenance
+  sessions:
+  - tasks:
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+- id: init
+  sessions:
+  - tasks:
+    - template_id: init
+- id: build
+  sessions:
+  - tasks:
+    - template_id: build
+- id: single_user_1_data_maintenance_1
+  sessions:
+  - tasks:
+    - template_id: single_user
+  - tasks:
+    - template_id: data_maintenance_delta
+    - template_id: data_maintenance_delta
+    connection_manager: 1
```

Comment:
clarification: all connection managers in this workload use the same connection manager, 1. Since this is a `concurrent_multi` workload, I was expecting at least 2 different connections.

Reply:
No, we use multiple connection managers, i.e., the connection manager is associated with the single session it is part of. If connection manager is not specified, connection manager `0` is used. I'll add further clarification in the form of comments.